concept/thing/attribute.feature

```diff
@@ -308,6 +308,19 @@ Feature: Concept Attribute
 
   Scenario: Attribute with value type double can be owned
 
-  Scenario: Attribute with value type string can be owned
+  Scenario: Attribute with value type datetime can be inserted in one timezone and queried in another timezone with no change in the value
```

Comment:
It looks like the file is arranged with creation first, then retrieval, then deletion.
Could you find where the scenario fits in and add it there instead of at the end of the file?